commands/pixl8/new/site.cfc

```diff
@@ -19,27 +19,82 @@ component {
 			return _printError( "Directory, [#arguments.directory#] is not empty." );
 		}
 
-		// scaffold the static dir
-		command( "pixl8 new static" ).run();
-
-		// create and scaffold the website dir
-		var webDir = ListAppend( arguments.directory, "website", "/" );
-		DirectoryCreate( webDir );
-		shell.cd( webDir );
-
-		command( "preside new site" ).params(
-			skeleton = "pixl8:pixl8-website-skeleton"
-		).run();
-
-		shell.cd( arguments.directory );
-
-		DirectoryCreate( arguments.directory & "/build" );
-		FileWrite(  arguments.directory & "/build/.gitignore", "*
-!.gitignore
-" );
-		FileMove( webDir & "/Dockerfile", arguments.directory & "/Dockerfile" );
-		FileMove( webDir & "/docker-compose.yml", arguments.directory & "/docker-compose.yml" );
-		FileMove( webDir & "/.gitlab-ci.yml", arguments.directory & "/.gitlab-ci.yml" );
+		//choose admin or site
+		var appType = "";
+
+		var appTypeList = "site,webapp,rest" 
+
+		print.line();
+		print.line( "Available site templates from which to build your new site/application:" );
+
+		print.text( " * " );
+		print.yellowText( "site" );
+		print.line( ": Pixl8 Website template." );
+
+		print.text( " * " );
+		print.yellowText( "webapp" );
+		print.line( ":  Web applications that are purely admin based." );
+
+		print.text( " * " );
+		print.yellowText( "rest" );
+		print.line( ": REST webservice application." );
+
+
+		while( !listFindNoCase( appTypeList , appType) ) {
+			appType = LCase(ask( "Enter site template: " ));
+		}
+		switch(LCase(appType)){
+
+		case "webapp":
+
+			var webDir = ListAppend( arguments.directory, "website", "/" );
+			
+			command( "preside new site" ).params(
+				skeleton = "preside-skeleton-webapp"
```

Comment:
We have a skeleton in Pixl8's private package repo. We should use that one.

Reply:
changed to use pxix8 admin site skeleton. pending the skeleton to be packaged